docs/framework/windows-workflow-foundation/how-to-update-the-definition-of-a-running-workflow-instance.md

```diff
@@ -1406,83 +1408,84 @@ Dynamic update provides a mechanism for workflow application developers to updat
 
 21. Press CTRL+SHIFT+B to build the solution, and then press CTRL+F5 to run the `ApplyDynamicUpdate` application and update the persisted workflow instances. You should see output similar to the following. The version 1.0.0.0 workflows are updated to version 1.5.0.0, while the version 2.0.0.0 workflows are not updated.
 
- **Inspecting: StateMachineNumberGuessWorkflow; Version=1.0.0.0**
-**Updated to: StateMachineNumberGuessWorkflow; Version=1.5.0.0**
-**Inspecting: StateMachineNumberGuessWorkflow; Version=1.0.0.0**
-**Updated to: StateMachineNumberGuessWorkflow; Version=1.5.0.0**
-**Inspecting: FlowchartNumberGuessWorkflow; Version=1.0.0.0**
-**Updated to: FlowchartNumberGuessWorkflow; Version=1.5.0.0**
-**Inspecting: FlowchartNumberGuessWorkflow; Version=1.0.0.0**
-**Updated to: FlowchartNumberGuessWorkflow; Version=1.5.0.0**
-**Inspecting: SequentialNumberGuessWorkflow; Version=1.0.0.0**
-**Updated to: SequentialNumberGuessWorkflow; Version=1.5.0.0**
-**Inspecting: SequentialNumberGuessWorkflow; Version=1.0.0.0**
-**Updated to: SequentialNumberGuessWorkflow; Version=1.5.0.0**
-**Inspecting: SequentialNumberGuessWorkflow; Version=1.0.0.0**
-**Updated to: SequentialNumberGuessWorkflow; Version=1.5.0.0**
-**Inspecting: StateMachineNumberGuessWorkflow; Version=1.0.0.0**
-**Updated to: StateMachineNumberGuessWorkflow; Version=1.5.0.0**
-**Inspecting: FlowchartNumberGuessWorkflow; Version=1.0.0.0**
-**Updated to: FlowchartNumberGuessWorkflow; Version=1.5.0.0**
-**Inspecting: StateMachineNumberGuessWorkflow; Version=2.0.0.0**
-**Inspecting: StateMachineNumberGuessWorkflow; Version=2.0.0.0**
-**Inspecting: FlowchartNumberGuessWorkflow; Version=2.0.0.0**
-**Inspecting: FlowchartNumberGuessWorkflow; Version=2.0.0.0**
-**Inspecting: SequentialNumberGuessWorkflow; Version=2.0.0.0**
-**Inspecting: SequentialNumberGuessWorkflow; Version=2.0.0.0**
-**Press any key to continue . . .**
+    **Inspecting: StateMachineNumberGuessWorkflow; Version=1.0.0.0**\
```

Comment:
Ideally, this would be inside a code block like the following

```markdown
```output
Inspecting: StateMachineNumberGuessWorkflow; Version=1.0.0.0
etc.
```

````

Reply:
Yeah, I left it as-is because I assumed that the bolding of the text was wanted